src/nxdoc/nuxeo-server/installation/tomcat-server-zip.md

```diff
@@ -122,3 +126,41 @@ Java packages and instructions for installation are available from the Azul webs
 As for any software, we very strongly recommend upgrading to the latest bugfix version of the JDK for any given major version
 
 {{! /multiexcerpt}}
+
+## General LTS 2021 Requirements
+
+{{{multiexcerpt 'lts2021-general-prerequesites' space='nxdoc' page='generic-multi-excerpts'}}}
```

Comment:
I think this section is still duplicating the above Requirements one.
This is what it looks like:
![image](https://user-images.githubusercontent.com/1265714/106497861-e0063880-64be-11eb-86af-27297bb28eb0.png)
and below:
![image](https://user-images.githubusercontent.com/1265714/106497877-e72d4680-64be-11eb-9799-c5f3cbe5853e.png)